packages/next/src/server/image-optimizer.ts

```diff
@@ -520,7 +520,17 @@ export async function optimizeImage({
   height?: number
 }): Promise<Buffer> {
   const sharp = getSharp()
-  const transformer = sharp(buffer).timeout({ seconds: 7 }).rotate()
+  const transformer = sharp(buffer, {
+    limitInputPixels: process.env.__NEXT_EXPERIMENTAL_IMAGE_PIXELS
+      ? Number(process.env.__NEXT_EXPERIMENTAL_IMAGE_PIXELS)
+      : 67_108_864,
```

Comment:
Would it make more sense for the non-experimental case to pass `undefined` for this input to prevent any change in behavior from the default set by `sharp`?

Reply:
Sure we can do that.

I changed it because the default in the latest sharp is 16383x16382, which is 268 megapixels ❗ 

So I figured a sane default would be 8192x8192 which is 67 megapixels since the largest photo an iPhone can take right now is 48 megapixels).

Reply:
Updated in [e3fa037](https://github.com/vercel/next.js/pull/71733/commits/e3fa0375c9d17c64ae012cefb376cca4b5a60332)